x/programs/rust/sdk_macros/tests/ui/ignore-second-param.rs

```diff
@@ -0,0 +1,12 @@
+use sdk_macros::public;
+
+// TODO:
+// We need to actually make sure this is the proper type
+// Currently, it's just a name check
+// and would break if you used a fully qualified name
```

Comment:
nit: should this TODO be moved or replicated at the implementation point in `is_program` in `lib.rs`?

Reply:
Haha great nit! There's proof that you're correct in that this is the wrong place for this TODO... I actually (kind of) fixed this after I wrote the TODO. The fix was obviously in `lib.rs`. 

You've also made me realize that I actually need to add another test for passing in a type that's called `Program` but isn't the same `wasmlanche_sdk::program::Program`

Reply:
fixed